netci.groovy

```diff
@@ -85,12 +86,11 @@ pushd %WORKSPACE%\\roslyn-internal
 git submodule init
 git submodule sync
 git submodule update --init --recursive
-init.cmd
 """)
 
             // Build the SDK and install .NET Core Templates.
             batchFile("""
-echo *** Build the SDK and install .NET Core Templates  ***
+echo *** Build the SDK  ***
```

Comment:
We are not using the SDK VSIXes, AFAIK. 1. the CSharp VSIXes cannot be installed because of a versioning problem. 2. We are not running VSI tests on the Basic VSIXes